spec/ics-027-interchain-account/README.md

```diff
@@ -0,0 +1,285 @@
+---
+ics: 27  
+title: Interchain Account  
+stage: Draft  
+category: IBC/TAO  
+author: Tony Yun <yunjh1994@everett.zone>, Dogemos <josh@tendermint.com>  
+created: 2019-08-01  
+modified: 2019-11-08  
+---  
+
+## **Synopsis**  
```

Comment:
I don't think you need both the header & bold, it will format correctly in a Markdown render with just '##'. (this comment also applies to other headers in this document)